### DISCUSSION TITLE
Proxmox: 7.3-3 Kernel: 6.1.0-1

### DISCUSSION THREAD

**Discussion by Aterfax** [Aterfax/relax-intel-rmrr#28] [Announcements]
Built for Proxmox 7.3-3 via the Docker image.

Tested and works for passing a Quadro through on my DL380P.

Name: release.zip
Size: 103386140 bytes (98 MiB)
SHA256: 57e119ce59a18082e152acc4b66c19284a5f4fdc0170d08b14cd6a4c64071a7c
SHA1: b8d391b15e6504fc375c095a08a0c89387a8dbb6
BLAKE2sp: 253681c9debe9914c6f0218c772c018d68cd3c532d71ab363f8aa78bf2191b35

This release and those going forward are signed with my GPG keys, see the following link for further details and the associated public key: https://github.com/Aterfax/relax-intel-rmrr/discussions/16

## What's Changed
* Add missing python dep. by @Aterfax in https://github.com/Aterfax/relax-intel-rmrr/pull/26

**Full Changelog**: https://github.com/Aterfax/relax-intel-rmrr/compare/v5.19.7-2...v6.1.0-1

<hr /><em>This discussion was created from the release <a href='https://github.com/Aterfax/relax-intel-rmrr/releases/tag/v6.1.0-1'>Proxmox: 7.3-3 Kernel: 6.1.0-1</a>.</em>

**Comment by Aterfax** [comment]
I'd advise people do a bit more testing than they normally would due to the major kernel version change. In any case, good luck, have fun and have a merry Xmas!

**Reply by Krushx0** [reply]
Could you release the updated deb to the current Proxmox version and kernel: 7.3-4  kernel:5.15.83-1-pve?

**Reply by Aterfax** [reply]
Hi there,

This release is ahead of 5.15.83. Did you want an older version? Previous releases can also be found here: https://github.com/Aterfax/relax-intel-rmrr/releases

Or if you don't see the one you want you can look through the auto builds here: https://github.com/Aterfax/relax-intel-rmrr/actions

If you look in the actions steps and review the Zip up debs step, you will see the packages for that action.

e.g.  
![image](https://user-images.githubusercontent.com/8648058/213263892-1dd6d145-3941-4cd4-8edc-1aa7293733b3.png)


If you wanted newer versions then you can do the same thing but look at the most recent package generated.
